test/test_matcher.py

```diff
@@ -219,8 +220,9 @@ def test_multiple_records(s3, sqs, mocker):
     assert res[1]["result"] == "testmatch"
 
 
-def test_bucket_not_found(s3, sqs, mocker):
-    with pytest.raises(s3.meta.client.exceptions.NoSuchBucket):
+@mock_s3
```

Comment:
Out of curiosity, what does this decorator do?

Reply:
Nothing at all :smile: I was trying to get the test to work but this didn't help. I've removed it.